profile/rwahs.xml

```diff
@@ -6116,6 +6116,9 @@
       <groupAccess>
         <permission group="admin" access="edit"/>
         <permission group="museum" access="edit"/>
+        <permission group="library" access="edit"/>
```

Comment:
These 3 lines were missing, as all roles have access to managing the subject list.